File: src/main/java/cn/pprocket/Logger.java
package cn.pprocket;

import java.time.LocalTime;
import java.time.format.DateTimeFormatter;

public class Logger {
    String className;
    public Logger(Object o) {
        className = o.getClass().getCanonicalName();
    }
    public Logger(String className) {
        this.className = className;
    }
    private String time() {
        LocalTime now = LocalTime.now();


        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm:ss");


        return now.format(formatter);

    }

    public void info(String message) {
        System.out.printf("[INFO] [%s] [%s] %s%n", time(), className, message);
    }
    public void error(String message) {
        System.out.printf("\033[31m"+ "[ERROR] [%s] [%s] %s%n", time(), className, message);
    }
}

File: src/commonMain/kotlin/cn/pprocket/Logger.kt
package cn.pprocket

import kotlinx.datetime.Clock
import kotlinx.datetime.TimeZone
import kotlinx.datetime.format
import kotlinx.datetime.format.DateTimeFormat
import kotlinx.datetime.toLocalDateTime

class Logger(var className: String) {


    private fun time(): String {
        val currentTime = Clock.System.now() // 获取当前时间
        val time = currentTime.toLocalDateTime(TimeZone.currentSystemDefault()).time


        return ""
    }

    fun info(message: String?) {
        println("[INFO] [${time()}] [%${className}] ${message}")
    }
    fun error(message: String?) {
        info(message)
    }


}
